dev-haskell/hmatrix/files/hmatrix-0.20.2.0-gentoo-blas.patch

```diff
@@ -0,0 +1,70 @@
+diff -urN hmatrix/packages/base/hmatrix.cabal hmatrix-r1/packages/base/hmatrix.cabal
+--- hmatrix/packages/base/hmatrix.cabal	2022-02-27 21:37:00.805021893 -0700
++++ hmatrix-r1/packages/base/hmatrix.cabal	2022-02-27 21:43:12.529392192 -0700
```

Comment:
There's something off about the paths here that's causing `patch` to fail. However, it works if I adjust it to this:

```diff
diff -urN hmatrix/hmatrix.cabal hmatrix-r1/hmatrix.cabal
--- hmatrix/hmatrix.cabal	2022-02-27 21:37:00.805021893 -0700
+++ hmatrix-r1/hmatrix.cabal	2022-02-27 21:43:12.529392192 -0700
```

Reply:
Rebuilt the patch.  I must have needed to be 1 directory level down from the cabal file.